### ISSUE TITLE
BUILD: Destroying MySQL connection when using dynamicDNS

### ISSUE THREAD

**Issue by nitro99x** [DarkflameUniverse/DarkflameServer#259]
### Make sure you've done the following:

- [X] I have read the [installation guide](https://github.com/DarkflameUniverse/DarkflameServer/blob/main/README.md).

### Platform

Ubuntu

### Architecture

x86

### Error Logs

<details>

Replace this line with the error logs.

</details>

**Comment by nitro99x** [comment]
Hello everyone,
I tried setting up my server using a ddns address (say darkflameserver.ddns.net for example) for my external ip in all three config files and the client. I kept running into Destroying MySQL connection whenever I tried to use it, so I ran it using my actual public ip4 and it worked. Is there a way for me to use a DDNS? That would be great, since I get a new public ip4 everyday. I know from my Minecraft server that the DDNS-address is working.
Thanks in advance
nitro99x
